azure/scope/cluster.go

```diff
@@ -284,7 +284,7 @@ func (s *ClusterScope) LBSpecs() []azure.ResourceSpecGetter {
 			SubnetName:        s.ControlPlaneSubnet().Name,
 			FrontendIPConfigs: []infrav1.FrontendIP{
 				{
-					Name: s.APIServerLB().Name + "-internal-frontEnd", // TODO: improve this name.
+					Name: s.APIServerLB().Name + "-internal-frontEnd",
```

Comment:
Because we didn't improve this name we probably can't remove the // TODO :)

Reply:
🥲

Reply:
Added back the TODO.